### ISSUE TITLE
Support vscode integrated terminal

### ISSUE THREAD

**Issue by nirnatan** [ranyitz/newsh#11]
Is it possible to add support for vscode integrated terminal? 
Currently the command opens 'Terminal' process with the split command

**Comment by ranyitz** [comment]
@nirnatan I couldn't find a way to do it yet. It needs some investigation.

I'll paste here issues that looks relevant:

[Launch task directly into split terminal](https://github.com/microsoft/vscode/issues/47265)

[How to run a script in a new Integrated Terminal window from an Integrated Terminal window](https://stackoverflow.com/questions/56430603/how-to-run-a-script-in-a-new-integrated-terminal-window-from-an-integrated-termi?rq=1)

[Provide commands to split the terminal](https://github.com/microsoft/vscode/issues/7504)

We can maybe find a workaround using `tasks` API.

**Comment by ranyitz** [comment]
Thanks @orta & @Tyriar! 

> Additionally we don't allow splitting directionally currently, it just splits depending on the orientation of the terminal

That makes sense to me.

> There would need to be a CLI API to split the terminal which doesn't exists and I don't think has come up before

I would love to open an issue if you think that it would be relevant.
__________

It seems that a similar API to the one suggested here is being developed on [windows terminal](https://github.com/microsoft/terminal/blob/master/doc/user-docs/UsingCommandlineArguments.md#using-the-wtexe-commandline). This can be handy for vscode's integrated terminal as well.

**Comment by Tyriar** [comment]
@ranyitz yes could you open an in vscode to add a cli? I think it would have to be a subgroup like `code terminal split-pane`, not just `code split-pane` as it would pollute the top-level namespace too much.

**Comment by ranyitz** [comment]
Thanks @Tyriar, issue opened - https://github.com/microsoft/vscode/issues/92031
